Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -2385,18 +2384,16 @@ script:
     - contextPath: SentinelOne.Notes.UpdatedAt
       description: The note updated time.
       type: string
-  dockerimage: demisto/python3:3.10.13.83255
+  dockerimage: demisto/python3:3.10.13.84405
   isfetch: true
   ismappable: true
   isremotesyncin: true
   isremotesyncout: true
   script: ""
   type: python
   subtype: python3
+  runonce: false
```

Comment:
I did not intentionally add this

Reply:
HI @randomizerxd , can you please revert this change